### PR TITLE
fix: Hardening “About Wire” window (WEBAPP-4478)

### PR DESCRIPTION
**Pull request by pwnsdx**
- Now using Electron sandbox, preload script, ipc, nodeIntegration on false and javascript is also disabled on rendered page
- Prevent dropping links / following any kind of links but
- Only allow https:// links to be opened in the user browser (Chrome, Edge, FF…)
- Prevent the window from being minimized/maximized
- Always put the window on top of the other windows
- Typos

### CODE REVIEW

File: electron/main.js [hunk 1]
```diff
@@ -24,12 +24,17 @@ const fs = require('fs-extra');
 const minimist = require('minimist');
 const path = require('path');
 const raygun = require('raygun');
-const { app, BrowserWindow, ipcMain, Menu, shell } = require('electron');
+const { session, app, BrowserWindow, ipcMain, Menu, shell } = require('electron');
```
Comment (lipis) [inline electron/main.js]: no bracket spacing and put the contents in alphabetical order

File: electron/main.js [hunk 2]
```diff
@@ -298,23 +304,83 @@ const showMainWindow = () => {
...
       height: config.WINDOW.ABOUT.HEIGHT,
+      maximizable: false,
+      minimizable: false,
       resizable: false,
-      fullscreen: false
+      show: false,
+      title: config.NAME,
+      width: config.WINDOW.ABOUT.WIDTH,
+      backgroundColor: '#ececec',
+      webPreferences: {
```
Comment (lipis) [inline electron/main.js]: sort all properties in alphabetical order

File: electron/main.js [hunk 3]
```diff
@@ -298,23 +304,83 @@ const showMainWindow = () => {
...
+        session: session.fromPartition('about-window'),
+        javascript: false,
+        nodeIntegration: false,
+        nodeIntegrationInWorker: false,
+        preload: path.join(APP_PATH, 'js', 'about.js'),
```
Comment (gregor) [inline electron/main.js]: If this is now a preload file, this needs to be reflected in the name.

File: yarn.lock [hunk 4]
```diff
@@ -0,0 +1,6619 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (gregor) [inline yarn.lock]: No yarn on wire-desktop

File: .gitignore [hunk 5]
```diff
@@ -14,3 +14,4 @@ app.asar
 node_modules
 npm-debug.log
 Wire*.zip
+yarn.lock
```
Comment (gregor) [inline .gitignore]: Alphabetical order if you really want to add it here.

**Pull request merged**